Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -1330,7 +1332,7 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
 
         const TypeIndex calleeTypeIndex = m_expressionStack.last().type().index;
         const TypeDefinition& typeDefinition = TypeInformation::get(calleeTypeIndex);
-        const auto& calleeSignature = *typeDefinition.as<FunctionSignature>();
+        const auto& calleeSignature = *typeDefinition.expand().as<FunctionSignature>();
```

Comment:
This `expand()` call was adjusted to match the `callIndirect` case above, so that it passes the `typeDefinition` unexpanded below.